### DISCUSSION TITLE
about vdf config set

### DISCUSSION THREAD

**Discussion by westfruit** [Chia-Network/chia-blockchain#6697] [Q&A]
vdf clients connect through HTTP to the timelord. So you can have the timelord in a separate machine as the timelord launcher

The chia config ymal show vdf_client and vdf_server should i set vdf_server to a remote farmer's ip and set vdf_client to localhost?

**Comment by isipisi89** [comment]
This is a good question. I want to run the Timelord on a separate machine too but whant to tell the full node/farmer my own timelord in local network. Somebody can help?
